test/change.test.js

```diff
@@ -82,6 +82,36 @@ describe('Change', function() {
     });
   });
 
+  describe('Change.rectifyModelChanges - promise variant', function() {
+    describe('using an existing untracked model', function() {
+      beforeEach(function(done) {
+        var test = this;
+        Change.rectifyModelChanges(this.modelName, [this.modelId])
+          .then(function(trackedChanges) {
+            done();
+          })
+          .catch(function(err) {
+            return done(err);
+          });
+      });
+
+      it('should create an entry', function(done) {
+        var test = this;
+        Change.find(function(err, trackedChanges) {
```

Comment:
AFAICT, this is using the old call-back based API. Please rework it to use promises.